scripts/generate_hyperparameters.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
We should add a basic description of the usage here and also mention this script in the main readme when we introduce hyperpararameters.  The usage message, or the readme in this directory, should tell users how to format the input files.

Reply:
The usage can also describe which command line arguments are accepted.  Currently this is only the `family`, but we can add `numreplicates` there as well.